openedx/adg/admin_dashboard/webinars/api/views.py

```diff
@@ -4,19 +4,20 @@
 from django.utils.decorators import method_decorator
 from django.views.decorators.csrf import ensure_csrf_cookie
 from rest_framework.generics import get_object_or_404
-from rest_framework.permissions import IsAuthenticated
 from rest_framework.response import Response
 from rest_framework.status import HTTP_200_OK
 from rest_framework.views import APIView
 
+from openedx.adg.admin_dashboard.permissions import IsADGAdmin
+from openedx.adg.common.lib.zoom_client.client import ZoomClient
 from openedx.adg.lms.webinars.models import Webinar
 
 
 class WebinarCloneView(APIView):
     """
     View to clone a webinar
     """
-    permission_classes = (IsAuthenticated,)
+    permission_classes = (IsADGAdmin,)
```

Comment:
I think we should keep `IsAuthenticated`. What if unauthenticated user tries to hit the API, Exception will be raised as per my understanding that anonymous user has no attribute `is_active`.

Reply:
I checked, its not raising.